meterfs/file_pc.c

```diff
@@ -0,0 +1,144 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs test file abstraction
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ * %LICENSE%
+ */
+
+#include <string.h>
+#include <host-flashsrv.h>
+#include <meterfs.h>
+
+#include "file.h"
+
+#define FLASHSIZE 4*1024*1024
+#define SECTORSIZE 4*1024
+
+int file_lookup(const char *name)
+{
+	id_t id;
+
+	return hostflashsrv_lookup(name, &id);
+}
+
+
+int file_open(const char *name)
+{
+	int err;
+	id_t id;
+
+	if ((err = hostflashsrv_lookup(name, &id)) < 0)
+		return err;
+
+	if ((err = hostflashsrv_open(&id)) < 0)
+		return err;
+
+	return (int)id; 
+}
+
+
+int file_close(int fid)
+{
+	id_t id = (id_t)fid;
```

Comment:
It will work in most cases, but we shouldn't really assume that `id_t` can be safely casted to `int`.

Reply:
Done.